templates/netlify/netlify.toml

```diff
@@ -1,17 +1,18 @@
 [build]
-  command = "remix build"
-  publish = "public"
+command = "remix build"
+publish = "public"
 
 [dev]
-  command = "remix watch"
-  port = 3000
+command = "remix watch"
+port = 3000
```

Comment:
We should be able to remove this @ascorbic since we have this info in https://github.com/netlify/framework-info/blob/main/src/frameworks/remix.json? I'll add a PR there to add the port though.

Reply:
Yeah, we can remove the whole `[dev]` section

Reply:
So keep the `build` section even though it's in framework info as well?